index.html

```diff
@@ -287,21 +294,28 @@ <h2>Linked Data Signature Overview</h2>
         <dt>type (required)<dt>
         <dd>
 A URI that identifies the digital <a>signature suite</a> that was used to create the
-signature. For example: <code>RsaSignature2018</code>.</dd>
-        <dt>creator (required)<dt>
-        <dd>
-A URI that identifies the public/private key pair associated with the
-signature. The URI SHOULD be a URL that can be dereferenced to obtain a
-<a>linked data document</a> that contains a link identifying the entity that
-owns the key pair. Dereferencing the entity link SHOULD result in a Linked Data
-document that contains a link back to the URL identifier for the
-public/private key pair, thereby proving ownership.
-        </dd>
+signature. For example: <code>Ed25519Signature2018</code>.</dd>
         <dt><dfn>created</dfn> (required)<dt>
         <dd>
 The string value of an [[!ISO8601]] combined date and time string generated
 by the <a href="#signature-algorithm">Signature Algorithm</a>.
         </dd>
+
+        <dt>proofPurpose (required)<dt>
```

Comment:
@dlongley - Does `proofPurpose` or `verificationMethod` belong in the LD Signatures spec?

Reply:
They belong in the LD Proofs spec.

Reply:
Thx, I'll remove them. (Should I still leave those fields in the various examples, since users are likely to encounter them as part of proofs?)

Reply:
Yes, leave them in examples -- and we'll want to make it clear to them where those things are coming from with appropriate text/links to the LD Proofs spec. Not sure how we want to do this, but we don't want people to be lost when they see those things.